File: eunji/0x03/BOJ_2577.py
A = int(input())
B = int(input())
C = int(input())
# print(A,B,C)

target = A*B*C
counter = {i:0 for i in range(10)}
# for i in range(10):
    # counter[i] = 0
    
while (target > 0):
    num = target % 10
    counter[num] += 1
    target = target // 10
    
for i in range(10):
    print(counter[i])

File: eunji/0x03_array/BOJ_2577.cpp
#include <iostream>
using namespace std;

int main(void){
    int A; int B; int C;
    cin >> A >> B >> C;
    int target = A*B*C;
    int counter[10] = {};
    while (target > 0){
        int num = target % 10;
        counter[num] ++ ;
        target = target / 10;
    }

    for (int i=0; i<10; i++){
        cout << counter[i] << '\n';
    }
}